ETC.tsv

```diff
@@ -7202,7 +7202,7 @@ ETC_20150317_007201	Elementalist Master
 ETC_20150317_007202	Doppelsoeldner Master
 ETC_20150317_007203	Baby Panto
 ETC_20150317_007204	Crystal Basket
-ETC_20150317_007205	Asleeped Nepenthes
+xETC_20150317_007205	Asleep Nepenthes
```

Comment:
If this line refers to a Nepenthe that is sleeping, it should read "Sleeping Nepenthe" rather than "Asleep Nepenthe".